test/models/role_test.rb

```diff
@@ -0,0 +1,28 @@
+require "test_helper"
+
+class RoleTest < ActiveSupport::TestCase
+  def setup
+    @organization = create(:organization)
+    @staff_account = create(:staff_account, organization: @organization)
+  end
+
```

Comment:
Since we've added https://github.com/thoughtbot/shoulda-matchers you can add some quick tests to make sure the validation & the belongs_to works.

Reply:
For my benefit, how is this not testing Rails code? I.e. per the model unit tests I hae below, we don't want them as they test the gem functionality, doesn't this test Rails associations functionality? I am happy to add them, but then I'd argue a case for keeping the tests that ensure roles are added and removed.

Reply:
Yeh you are right they are close in similarity. I think these shoulda validations & associations are very low cost that it just checks that you included those lines of code in your model. For the low price of having it, I generally just include them in case a person wants to remove a validation unknowningly. I would think differently if they cost more lines of code to have them.

Reply:
yes that is fine by me, I figured the same and added them.